wp-content/plugins/pri-migration-helper/migration/taxonomy_filters/taxonomy-category.php

```diff
@@ -243,8 +243,12 @@ function pri_fgd2wp_pre_insert_taxonomy_term( $custom_field_values, $new_term_id
 	switch ( $custom_field_name ) {
 		case 'category-body':
 			if ( $custom_field_values && ! empty( $custom_field_values[0] ) ) {
-				$args['description'] = get_term_field( 'description', $new_term_id, 'category', 'raw' ) . ' ' . $custom_field_values[0]['field_body_value'];
-				wp_update_term( $new_term_id, 'category', $args );
+				$cat_description = get_term_field( 'description', $new_term_id, 'category', 'raw' );
+				if ( is_string( $cat_description ) && empty( trim( $cat_description ) ) && ! empty( $custom_field_values[0]['field_body_value'] ) ) {
```

Comment:
This condition looks like if would fail if `$cat_description` was returned as anything other than a string, but we want this condition to succeed in that case. How about something like:

```
$cat_description_is_valid = is_string( $cat_description ) && ! empty( trim( $cat_description ) );
if ( ! $cat_description_is_valid && ! empty( $custom_field_values[0]['field_body_value'] ) ) {
  $args['description'] = $custom_field_values[0]['field_body_value'];
	wp_update_term( $new_term_id, 'category', $args );
}
```